setup_py.patch

```diff
@@ -1,14 +1,15 @@
---- setup.py.orig	2021-03-02 22:04:09.279820600 +0000
-+++ setup.py	 22:04:32.263841924 +0000
-@@ -90,9 +90,9 @@
+--- a/lib/py/setup.py
++++ b/lib/py/setup.py
```

Comment:
with this change I think you also need to add `-p1` to https://github.com/reddit/thrift-python/blob/0aaba41b1c62d8c620ead507bcb676471a926c3a/.github/workflows/python-publish.yaml#L30? (`-p1` is the short version of `--strip=1` which means it should strip one level of file name, e.g. the `a`/`b` level).

but maybe `patch` can guess that automatically?

Reply:
The CI run seems to have worked fine. I generated this output by making the desired change in the actual upstream repo and the running `git -p diff lib/py/setup.py` and copying the output to this file.